CorsixTH/Lua/entities/humanoid.lua

```diff
@@ -235,35 +235,39 @@ pee_anim("Invisible Patient",          4208)
 pee_anim("Transparent Female Patient", 4852)
 pee_anim("Transparent Male Patient",   4848)
 
+-- Some icons should only appear when the player hovers over the humanoid
+-- Higher priority is more important.
 --   | Available Moods |
 --   | Name            |Icon|Priority|Show Always| Notes
 -----+-----------------+----+--------+-----------+
-moods("reflexion",      4020,       5)            -- Some icons should only appear when the player
-moods("cantfind",       4050,       3)            -- hover over the humanoid
-moods("idea1",          2464,      10)            -- Higher priority is more important.
+moods("reflexion",      4020,       5)
+moods("cantfind",       4050,       3)
+moods("idea1",          2464,      10)
 moods("idea2",          2466,      11)
 moods("idea3",          4044,      12)
 moods("staff_wait",     4054,      20)
 moods("tired",          3990,      30)
 moods("pay_rise",       4576,      40)
 moods("thirsty",        3986,       4)
 moods("cold",           3994,       0,       true) -- These have no priority since
-moods("hot",            3988,       0,       true) -- they will be shown when hovering
-moods("queue",          4568,      70)             -- no matter what other priorities.
+moods("hot",            3988,       0,       true) -- they will be shown when
+                                                   -- hovering
+moods("queue",          4568,      70)             -- no matter what other
+                                                   -- priorities.
```

Comment:
I'd probably move the `true)` a bit to the left so you have enough room.
Priority numbers could move to the left as well, if desired.